### ISSUE TITLE
Crashes when trying to create a search folder

### ISSUE THREAD

**Issue by nekohayo** [lwindolf/liferea#307]
Liferea 1.10.17 (64 bits version currently in Fedora 23) crashes when you right-click in the feed list and choose to create a new search folder.

I wanted to get a gdb backtrace for you but liferea refuses to start (maybe because it's too slow?):

```
(gdb) set pagination 0
(gdb) run
Starting program: /usr/bin/liferea 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib64/libthread_db.so.1".
Detaching after fork from child process 24708.
[New Thread 0x7fffdc4ee700 (LWP 24709)]
[New Thread 0x7fffdbced700 (LWP 24710)]
[New Thread 0x7fffdb4ec700 (LWP 24711)]
[New Thread 0x7fffd8bf7700 (LWP 24712)]
[Thread 0x7fffd8bf7700 (LWP 24712) exited]
Failed to register: Le délai d'attente est dépassé (this translates to: (Wait limit/timeout reached")
[Thread 0x7fffdb4ec700 (LWP 24711) exited]
[Thread 0x7fffdbced700 (LWP 24710) exited]
[Thread 0x7ffff7f14a80 (LWP 24701) exited]
[Inferior 1 (process 24701) exited with code 01]
```

The Fedora automated crash catcher was able to trace it though, so here it is: https://bugzilla.redhat.com/show_bug.cgi?id=1298426

**Comment by yaneti** [comment]
The official 1.10.17 release tarball doesn't have search_folder.ui. This was fixed in 8b4ac3576c08fc7904a251e2219ca36ceb7fe4b0.

I think it warrants a new release..

**Comment by GreenLunar** [comment]
Does it happen with [Liferea 1.10.18)](https://github.com/lwindolf/liferea/releases/tag/v1.10.18)?

**Comment by yaneti** [comment]
On the Fedora side this should be fixed by https://bodhi.fedoraproject.org/updates/FEDORA-2016-3e291b9f03   - the F23 update with 1.10.18

**Comment by lwindolf** [comment]
This was fixed in 1.10.18 already.
